dpnp/backend/kernels/dpnp_krnl_common.cpp

```diff
@@ -655,10 +664,26 @@ void dpnp_matmul_c(void* result_out,
                    const shape_elem_type* input2_strides)
 {
     DPCTLSyclQueueRef q_ref = reinterpret_cast<DPCTLSyclQueueRef>(&DPNP_QUEUE);
-    dpnp_matmul_c<_DataType>(q_ref,
-                             result_out, result_size, result_ndim, result_shape, result_strides,
-                             input1_in, input1_size, input1_ndim, input1_shape, input1_strides,
-                             input2_in, input2_size, input2_ndim, input2_shape, input2_strides);
+    DPCTLEventVectorRef dep_events_ref = nullptr;
```

Comment:
`dep_event_vec_ref` is more descriptive or more appropriate than `dep_events_ref`